python/processors/Stop0l_postproc.py

```diff
@@ -169,7 +169,7 @@
 }
 
 #cut on raw resolved candidates, not final analysis cut
-DeepResovledCandidateDiscCut = 0.75
+DeepResovledCandidateDiscCut = 0.00
```

Comment:
I understand that the cut on deepresolved is moved to TopTagger. Should we either keep the cut value here or  just remove it? 0.00 value could be confusing for future readers.

Reply:
maybe we can just add a comment to explain.  And it was not, but removed, because we need all the tops in order to get the SF and systematic